sources/packages/web/src/views/student/StudentAccountActivity.vue

```diff
@@ -10,7 +10,7 @@
         :routeLocation="{ name: StudentRoutesConst.STUDENT_DASHBOARD }"
       />
     </template>
-    <formio formName="studentaccountactivity" :data="initialData"></formio>
+    <formio-container formName="studentAccountActivity" :data="initialData" />
```

Comment:
I have a feeling that this is the PR actually using the `formio-container` mostly for these cases.

Reply:
haha...in my institution pr also replaced `formio` with `formio-container`

Reply:
As e talked, there was no agreement and there is no real benefit (IMO) to using the  `formio-container`. As we also talked about, it will be up to the dev to choose.